tests/functional/object/mcg/test_bucket_deletion.py

```
@@ -123,11 +124,29 @@ def test_bucket_delete_with_objects(
         Negative test with deletion of bucket has objects stored in.
 
         """
-        bucketname = bucket_factory(bucketclass=bucketclass_dict)[0].name
+        logger.info(
+            f"Creating bucket with interface={interface}, "
+            f"bucketclass_dict={bucketclass_dict}"
+        )
+        bucket = bucket_factory(bucketclass=bucketclass_dict)[0]
+        bucketname = bucket.name
+        logger.info(f"Bucket {bucketname} created successfully")
```

Ruff (0.15.15)

[warning] Logging statement uses f-string

(G004)